src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocLexer.g4

```diff
@@ -37,8 +37,7 @@ import java.util.*;
 
 }
 
-LEADING_ASTERISK : ( (' '|'\t') {_tokenStartCharPositionInLine == 0}? ) (' '|'\t')* '*'
-      | '*' {_tokenStartCharPositionInLine == 0}?
+LEADING_ASTERISK : (' '|'\t')* '*'
```

Comment:
@rnveach Actually the tests are spit out by [this](https://github.com/checkstyle/contribution/blob/master/javadoc-expected-tree-generator/ExpectedParseTreeGenerator.java) in contribution as you might know. So I was just sticking to it. All this should be resolved in [4381](https://github.com/checkstyle/checkstyle/issues/4381).